### ISSUE TITLE
Content for what is ODH section

### ISSUE THREAD

**Issue by Gkrumbach07** [opendatahub-io/opendatahub.io#103]
## Objective
Edit, review, and suggest changes to the content below as a comment to this issue. Use the image attached as a reference of what the content will be rendered like. If content and wording looks good. Give a `lgmt` (looks good to me) in the comments of this issue, otherwise leave your suggestions.

## Current Content

**Title**: What is Open Data Hub?

- **Option 1:**  The Open Data Hub is a comprehensive collection of open source tools designed to facilitate the entire data science life cycle. The project was developed to take advantage of[ Kubernetes](https://kubernetes.io/) running on the Red Hat[ OpenShift® Container Platform](https://www.openshift.com/). It builds on upstream initiatives such as[ Data Science Pipelines](https://github.com/kubeflow/kfp-tekton/) and[ Model Mesh Serving](https://github.com/kserve/modelmesh-serving/) to serve as the foundation for Red Hat's internal data science and AI platform. Data scientists can use popular tools such as[ PyTorch](https://pytorch.org/) and[ TensorFlow™](https://www.tensorflow.org/), to create and develop models using Jupyter notebooks with ease. This allows teams to spend more time solving critical business needs rather than wasting valuable time installing and maintaining infrastructure.
The Open Data Hub strives to encourage collaboration between data scientists and other stakeholders, while following open source best practices

- **Option 2**: Red Hat Open Data Hub is a comprehensive platform for building, deploying, and managing data-intensive applications on Kubernetes. It is an open-source platform that offers a collection of integrated tools and technologies to help organizations accelerate their digital transformation initiatives by leveraging the power of big data, machine learning, and artificial intelligence.

- **Option 3**: The Open Data Hub is an all-encompassing platform, providing a diverse set of open source tools specifically engineered to streamline the data science life cycle. Developed with Kubernetes on the Red Hat OpenShift® Container Platform, this innovative project optimizes the construction, deployment, and management of data-intensive applications. As an open-source AI platform, the Open Data Hub furnishes a collection of cohesive tools and technologies that enable organizations to expedite their digital transformation initiatives by harnessing the potential of big data, machine learning, and artificial intelligence.

- **Option 4**: OpenDataHub.io is a comprehensive open-source platform designed to streamline the data science lifecycle by providing a suite of integrated tools for building, deploying, and managing data-intensive applications on Kubernetes and the Red Hat OpenShift® Container Platform. Developed to enhance collaboration between data scientists and stakeholders, the platform leverages big data, machine learning, and artificial intelligence, allowing users to focus on addressing critical business needs while simplifying infrastructure management and maintenance.

Design Example
---
<img width="862" alt="image" src="https://user-images.githubusercontent.com/12587674/234283084-cb6bd37b-6807-4a76-90a3-55d016644451.png">

**Comment by LaVLaS** [comment]
I think some combination of option 2 and 3 would be best for this section

**Comment by shgriffi** [comment]
I moved this to me.  I'll work on a revision to this shortly.

**Comment by shgriffi** [comment]
"Open Data Hub is an open source AI platform designed for the hybrid cloud.  The community seeks to bridge the gap between application developers, data stewards and data scientists by blending the leading open source AI tools with a unifying and intuitive user experience.  Open Data Hub reduces the time it takes to infuse applications, services or business operations with AI.  With [Kubernetes](https://kubernetes.io/), [OpenShift](https://www.redhat.com/en/technologies/cloud-computing/openshift) and [operators](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/) at its core, Open Data Hub also simplifies deployments at scale to private data centers, cloud providers, edge devices and anywhere in between."

@Gkrumbach07 does this work?
